cert-rotate/cert-rotate.py

```diff
@@ -0,0 +1,300 @@
+import pytz
+import json
+import requests
+import datetime
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from google.oauth2 import service_account
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+
+
+# Google IoT core project settings
+PROJECT_ID = "project-id"
```

Comment:
This needs to be `os.getenv('PROJECT_ID')` or similar (and the same for all configs below.

Reply:
OS Environment? Or the config file would be better?

Reply:
Environment is basically the new standard way of doing these. In particular if you'd run this inside say a dockerized environment (which would be advised).

Reply:
Moved to os environment variables